packages/containers/src/FilterBar/FilterBar.selectors.js

```diff
@@ -0,0 +1,53 @@
+import { List } from 'immutable';
+
+const CONTAINER_FILTER = 'Container(FilterBar)';
```

Comment:
that is the displayName of the component you should import it from the component